docs/cloud/pouta/persistent-volumes.md

```diff
@@ -134,6 +160,14 @@ the virtual machine to avoid data loss!**
 openstack server remove volume <server> <volume>
 ```
 
+If you want to delete a volume and the data contained on it you can do. The data will be deleted for ever, it will not be recoverable.
```

Comment:
"If you want to delete a volume and the data contained on it you can do." <- probably something is missing in this sentence

Reply:
It is fixed, I needed to remove the sentence that was also repeated below.